src/cljs/athens/types/default/view.cljs

```diff
@@ -189,7 +191,56 @@
 
 
   (zoomed-in-view
-    [_this _block-data _callbacks])
+    [_this block-data callbacks]
+    (let [{:block/keys [uid
```

Comment:
Please move it to dedicated `defn` above protocol definition.
When it's inline, protocol implementation grows big very fast.